development/image-syncer/external-images.yaml

```
@@ -45,7 +45,7 @@ images:
 - source: "quay.io/prometheus/alertmanager:v0.24.0"
 - source: "quay.io/prometheus/prometheus:v2.39.1"
 - source: "nats@sha256:bf9cae35d4496a6557faaaa46222bc55961ffdea0f6d833968a7fe798581a289"
```

You also need to update the image SHA
```suggestion
- source: "nats@sha256:b961cb563a3c05e116ece911ad75c0dca8580521e1dc744f63fa78b2543e0aa8"
```

---

thanks! 
btw why do we need the sha only for some images?